webpack.js

```diff
@@ -0,0 +1,18 @@
+const { merge } = require('webpack-merge')
+const path = require('path')
+const webpackConfig = require('@nextcloud/webpack-vue-config')
+
+const config = {
+	entry: {
+		admin: path.join(__dirname, 'src', 'admin.js'),
+		user: path.join(__dirname, 'src', 'user.js'),
+		public: path.join(__dirname, 'src', 'public.js'),
```

Comment:
Don't forget to update the controllers to inject the proper script name as it changed

Reply:
AH no, strange
https://github.com/nextcloud/terms_of_service/blob/a0180066d6c19cf78bb5999ef878020ed3f52da9/lib/AppInfo/Application.php#L80
https://github.com/nextcloud/terms_of_service/blob/a0180066d6c19cf78bb5999ef878020ed3f52da9/lib/AppInfo/Application.php#L83

Where is the admin one used though?

Reply:
ok found https://github.com/nextcloud/terms_of_service/blob/6cffa5954ed1089aa161bacb29ffe81d40c08dd6/templates/settings.php#L22

Reply:
it didn't thats why I overwrote it

Reply:
Yep I finally understood why you did it :p